spec/models/conservation_status_spec.rb

```diff
@@ -57,6 +57,26 @@
       status = ConservationStatus.make!( status: "NE", iucn: Taxon::IUCN_NOT_EVALUATED, user: nil )
       expect( status.geoprivacy ).to eq Observation::OPEN
     end
+
+    it "should reassess observations of taxon if obscuring" do
+      species = Taxon.make!( rank: Taxon::SPECIES )
+      Delayed::Job.delete_all
+      stamp = Time.now
+      cs = ConservationStatus.make!( taxon: species )
+      jobs = Delayed::Job.where("created_at >= ?", stamp)
+      expect(jobs.select{|j| j.handler =~ /reassess_coordinates_for_observations_of/m}).not_to be_blank
+    end
+
+    it "should not reassess observations of taxon if open and global" do
+      species = Taxon.make!( rank: Taxon::SPECIES )
+      Delayed::Job.delete_all
+      stamp = Time.now
+      cs = ConservationStatus.make!( taxon: species, geoprivacy: "Open" )
```

Comment:
Use `Observation::OPEN` here instead of the string literal. Testing against `"Open"` is testing against a value that doesn't exist in reality. You should probably also test the situation where geoprivacy is nil.